fileAnalyzer.cpp

```diff
@@ -1,7 +1,4 @@
 #include "fileAnalyzer.h"
-#include <ctime>
-#include <regex>
-#include <thread>
 
 
 void fileAnalyzer::parse(const std::string filePath)
```

Comment:
It is better to use bool return value for fileAnalyzer::parse

Reply:
I do not understand why do we need mutex in parse method. We call parse with different args (each call for different file) and this method only reads from file.